addons/mail/static/src/core/web/thread_model_patch.js

```diff
@@ -18,15 +18,20 @@ const threadPatch = {
                 r.remove();
             },
         });
+        this.isDisplayedInDiscussAppDesktop = fields.Attr(undefined, {
+            compute() {
+                if (this.store.discuss.isActive && !this.store.env.services.ui.isSmall) {
+                    return this.eq(this.store.discuss.thread);
+                }
+                return false;
+            },
+        });
```

Comment:
Note that could type `this` on the `compute` function so that the `this` is properly type.
```js
/** @this {import("models").Thread} */
```